kafka-consumer/src/main/java/com/networknt/kafka/consumer/KafkaConsumerState.java

```diff
@@ -315,7 +315,7 @@ public synchronized void seek(ConsumerSeekRequest request) {
                   new OffsetAndMetadata(partition.getOffset(), partition.getMetadata()));
           break;
         } catch (IllegalStateException ie) {
-          logger.info("seeking to topic = " + partition.getTopic() + " partition = " + partition.getPartition() + " offset = " + partition.getOffset() + " caught illegal state exception will sleep for 1 sec and retry again.");
+          logger.info("seeking to topic = " + partition.getTopic() + " partition = " + partition.getPartition() + " offset = " + partition.getOffset() + " caught illegal state exception will sleep for 1 sec and retry again. Exception is :: " + ie);
```

Comment:
We need the stacktrace. so we cannot concat the exception. We need to add a comma and then the ie.